exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,76 @@
+{
+   "#": [
+     "Not sure what to type here, come back to this later."
```

Comment:
if there is nothing in particular to add (any notes for anyone who wishes to port this exercise to a track), the entire thing can be deleted.